pkg/apis/kudo/v1beta1/instance_types.go

```diff
@@ -528,7 +519,7 @@ func remove(values []string, s string) (result []string) {
 // hasn't been added yet, the instance has a cleanup plan and the cleanup plan
 // didn't run yet. Returns true if the cleanup finalizer has been added.
 func (i *Instance) TryAddFinalizer() bool {
-	if !contains(i.ObjectMeta.Finalizers, instanceCleanupFinalizerName) {
+	if !funk.ContainsString(i.ObjectMeta.Finalizers, instanceCleanupFinalizerName) {
```

Comment:
Typed methods, e.g. `ContainsString, ContainsInt` do not use reflection and have the same performance as the replaced implementation.